src/js/uswds-init.js

```diff
@@ -1,35 +1,23 @@
-const docElem = window.document.documentElement;
-const loadingClass = "usa-js-loading";
 
-function addLoadingClass() {
-  docElem.className += ` ${loadingClass}`;
-}
+/* eslint-disable no-var */
+(function uswdsInit () {
+  var loadingClass = "usa-js-loading";
```

Comment:
Since this file isn't processed at all now, it might be worth considering whether to opt-in to [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) manually. This was previously added automatically by Babel ([example](https://unpkg.com/uswds@2.11.2/dist/js/uswds-init.js)).

```suggestion
(function uswdsInit () {
  "use strict";
  var loadingClass = "usa-js-loading";
```

Pro: Benefits as listed in above MDN article
Con: 13 more bytes 😄